doc-Installing_on_OpenShift_Container_Platform/topics/post-installation.adoc

```diff
@@ -145,6 +145,133 @@ $ oc create -f cfme-restore-job.yaml
+
+To deploy new container images for {product-title_short}, delete the pods, update the images, then rebuild the updated pods:
+
+. Scale the application StatefulSets to 0 replicas:
++
+----
+$ oc scale statefulset cloudforms --replicas=0
+$ oc scale statefulset cloudforms-backend --replicas=0
+----
++
+. Patch in the new version of the images:
++
+----
+$ oc patch statefulset cloudforms -p '{"spec": {"template": {"spec": {"containers": [{"name": "cloudforms", "image": "registry.redhat.io/cloudforms46/cfme-openshift-app-ui:<new-version-tag>"}]}}}}'
+$ oc patch statefulset cloudforms-backend -p '{"spec": {"template": {"spec": {"containers": [{"name": "cloudforms", "image": "registry.redhat.io/cloudforms46/cfme-openshift-app:<new-version-tag>"}]}}}}'
+----
++
+. Scale the StatefulSets back to your previous values to redeploy the pods:
++
+----
+$ oc scale statefulset cloudforms --replicas=<old-value>
+$ oc scale statefulset cloudforms-backend --replicas=<old-value>
+----
+
+Your {product-title_short} environment now updated.
+
+
+[[modified-template-update]]
+==== Updating the {product-title_short} Container Images and Template
+
+To update a {product-title_short} deployment with a modified template, update the container images and the template using the following steps:
+
+. Back up secrets, such as database encryption keys and credentials:
++
+[IMPORTANT]
+====
+Be careful to back up secrets in a secure location.
+====
++
+----
+$ oc export secret cloudforms-secrets > my_secrets.yml
+----
++
+. Scale the application StatefulSets to 0 replicas:
++
+----
+$ oc scale statefulset cloudforms --replicas=0
+$ oc scale statefulset cloudforms-backend --replicas=0
+----
++
+. Patch in the new version of the images:
++
+----
+$ oc patch statefulset cloudforms -p '{"spec": {"template": {"spec": {"containers": [{"name": "cloudforms", "image": "registry.redhat.io/cloudforms46/cfme-openshift-app-ui:<new-version-tag>"}]}}}}'
+$ oc patch statefulset cloudforms-backend -p '{"spec": {"template": {"spec": {"containers": [{"name": "cloudforms", "image": "registry.redhat.io/cloudforms46/cfme-openshift-app:<new-version-tag>"}]}}}}'
+----
++
+. Apply the changes to the project:
++
+[IMPORTANT]
+====
+* If you customized any parameters when originally deploying the application (parameters used with the `oc new-app` command in xref:deploying-the-appliance[]), you must set the same values in the `oc process` command here.
+
+ifdef::cfme[]
+* If you are updating a {product-title} 4.6.0 environment to version 4.6.1, update the `postgresql` ConfigMap mount path from `postgresql-config` to `postgresql-cfg`.
+endif::cfme[]
+
+ifdef::miq[]
+* If you are updating a {product-title} 5.9.0 environment to version 5.9.1, update the `postgresql` ConfigMap mount path from `postgresql-config` to `postgresql-cfg`.
+endif::miq[]
+====
++
+----
+$ oc process -p APPLICATION_REPLICA_COUNT=0 -l app=cloudforms,template=cloudforms -f cfme-template.yaml | oc replace -f -
```

Comment:
This is specifically the template for users using the podified database (rather than one external to the openshift cluster), do we want to generalize the template file name here? All the other steps should be the same.

Reply:
Good catch -- I've updated this step, and the command in two places (template name, and .yaml file) to match what's documented in the section  "1.3.1. Deploying the CloudForms Appliance Using an External Database". Does that look right?